code/script.js

```diff
@@ -32,16 +41,145 @@ const showMessage = (message, sender) => {
+  But since my employers were instructed to cheer people up in english – they hired me.`, 'bot')
+  // Just to check it out, change 'bot' to 'user' here ðŸ‘†
+  setTimeout(() => showMessage(`How old are you?`, 'bot'), 3000)
+}
+
+
+
+// QUESTION 3
+const grapeResponse = (event) => {
+showMessage("Such a sad story. He was later pressed into it anyway..", 'bot')
+inputWrapper.innerHTML = `
+<button id="omgButton">Omg, pour kid!</button>
+<button id="riesling">I guess everything happens for a riesling</button>
+`
+document.getElementById("omgButton").addEventListener('click', () => showMessage("Omg, pour kid!", 'user'))
+document.getElementById("riesling").addEventListener('click', () => showMessage("I guess everything happens for a riesling", 'user'))
+setTimeout(() => titanicQuestion(event), 5000)
+}
+
+
+// Question 2
+const grapeQuestion = (event) => {
+  showMessage("Have you heard about the baby grape that didn't want to be made into wine?", 'bot')
+  inputWrapper.innerHTML = `
+  <button id="yesButton">YES</button>
+  <button id="noButton">NO</button>
+`
+  document.getElementById("yesButton").addEventListener('click', () => showMessage("Yes", 'user'))
+  document.getElementById("noButton").addEventListener('click', () => showMessage("No", 'user'))
+setTimeout(() => grapeResponse(event), 5000)
+}
+
+// Question 1
+const handleAgeInput = (event) => {
+event.preventDefault()
+const userAge = ageInput.value
+showMessage(userAge, 'user')
+ageInput.value = ""
+setTimeout(() => showMessage(`Ah ok, ${userAge} and getting older huh. Well here's a wine joke that only gets better with age...`, 'bot'), 2000)
+setTimeout(() => grapeQuestion(event), 5000)
+}
+
+//QUESTION 4
+const titanicQuestion = (event) => {
+
+  setTimeout(() => showMessage("Uhm... Yeah.. So what's your name? Titanic?", 'bot'), 1000)
+  setTimeout(() => getNamething(event), 1000)
+
+}
+
+//INPUT FOR QUESTION 4
+const getNamething = (event) => {
+  inputWrapper.innerHTML = `
+  <div class="input-wrapper2" id="input-wrapper2">
+  <form id="name-form2">
+    <input id="name-input2" type="text" />
+    <button id="sendName" class="send-btn2" type="submit">
+      Send
+    </button>
+  </form>
+</div>
+  `
+
+  document.getElementById('name-form2').addEventListener('submit', handleNameInput)
+
 }
 
+// NAME INPUT HANDLING
+const handleNameInput = (event) => {
+
+  const nameInput = document.getElementById('name-input2')
+
+  event.preventDefault();
+  const userName = nameInput.value
+  showMessage(userName, 'user')
+  nameInput.value = ""
+  setTimeout(() => showMessage
+  (`So sorry ${userName}, that was a bad ice breaker.. I am a bit nervous about this job. I am just an actress after all.`, 'bot'), 2000)
+  setTimeout(() => showMessage
+  (`Anyway, now that we are talking about famous love stories.. Do you know what Jay-Z called Beyoncé after their engagement?`, 'bot'), 5000)
+  setTimeout(() => beyonceResponse(event), 5000)
+}
+
+// QUESTION 5
+const beyonceResponse = (event) => {
```

Comment:
You're not using the `event` as far as I can see, so no need to pass it on to the next function